.github/renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "config:best-practices",
+    ":gitSignOff",
+    ":rebaseStalePrs",
+    "group:allNonMajor",
+    "group:linters",
+    "group:test",
+    ":preserveSemverRanges",
+    ":pinOnlyDevDependencies"
+  ],
+  "labels": ["kind/dependency upgrade"],
```

Comment:
This kind prefix, is that a common pattern?

Reply:
not sure -- do you like this?

Reply:
No strong opinions at all, just haven't seen it. If there's prior art and we think there'll be some "classes" of labels to distinguish, then let's go

Reply:
leaving it as is for now, unless a better idea comes up :D